cargo-concordium/src/build.rs

```diff
@@ -103,20 +103,7 @@ pub fn build_contract(
     let manifest = Manifest::from_path("Cargo.toml").context("Could not read Cargo.toml.")?;
     let package = manifest.package.context("Manifest needs to specify [package]")?;
 
-    let result = Command::new("cargo")
-        .arg("build")
-        .args(&["--target", "wasm32-unknown-unknown"])
-        .args(&["--release"])
-        .args(&["--target-dir", "target/concordium"])
-        .args(cargo_args)
-        .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
-        .output()
-        .context("Could not use cargo build.")?;
-
-    if !result.status.success() {
-        anyhow::bail!("Compilation failed.")
-    }
+    build_contract_with_cargo(cargo_args, false, true)?;
```

Comment:
Shouldn't this be more like?

```suggestion
    // emit the build log here iff. we have not already done so.
    build_contract_with_cargo(cargo_args, false, build_schema.build())?;
```
So if the schema is enabled then the build log is already emitted via `build_contract_schema` and we set quiet to true.
If the schema is not enabled we need to set `quiet` to `false` as we want to build now and emit logs.